configs/common/packages.yaml

```diff
@@ -91,6 +91,8 @@
       # Note: Uncommenting this entry will break
       # the container builds.
       #version: [2.11.0]
+    glib:
```

Comment:
What do we need this for in the ESMF variant PR? If I had to guess I'd say this will break the builds on more than just one platform.

Reply:
I don't remember adding this, I'll have to see where this came from...